genie-web/src/main/java/com/netflix/genie/web/properties/AgentCleanupProperties.java

```diff
@@ -48,5 +48,7 @@
 
     private long refreshInterval = 10_000;
 
-    private long timeLimit = 120_000;
+    private long reconnectTimeLimit = 120_000;
```

Comment:
Something i've wanted to do for a while and maybe you can start doing it here is to leverage the better property conversion stuff for more readable/flexible durations. See [here](https://docs.spring.io/spring-boot/docs/2.2.7.RELEASE/reference/html/spring-boot-features.html#boot-features-external-config-conversion-duration)

They also have converters for [data sizes](https://docs.spring.io/spring-boot/docs/2.2.7.RELEASE/reference/html/spring-boot-features.html#boot-features-external-config-conversion-datasize)

Reply:
Cool! Yes i'll convert this and use these going forward.